app/addons/custom_carriers/func.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/***************************************************************************
+ *                                                                          *
+ *   (c) 2004 Vladimir V. Kalynyak, Alexey V. Vinokurov, Ilya M. Shalnev    *
+ *                                                                          *
+ * This  is  commercial  software,  only  users  who have purchased a valid *
+ * license  and  accept  to the terms of the  License Agreement can install *
+ * and use this program.                                                    *
+ *                                                                          *
+ ****************************************************************************
+ * PLEASE READ THE FULL TEXT  OF THE SOFTWARE  LICENSE   AGREEMENT  IN  THE *
+ * "copyright.txt" FILE PROVIDED WITH THIS DISTRIBUTION PACKAGE.            *
+ ****************************************************************************/
+
+defined('BOOTSTRAP') or die('Access denied');
+
+/**
+ * Displays information about the module on the module settings page
+ *
+ * @return string
+ */
+function fn_custom_carriers_info()
+{
+    $text = __('custom_carriers.general_info');
+
+    return $text . '<hr/>' ;
+}
+
+/**
+ * Get an array of custom carriers in the same form as Shippings::getCarriers()
+ *
+ * @param string $is_custom For select custom or not custom carriers
+ *
+ * @return array<string, array<string, string>> $normalize_carriers Normalize custom carriers
+ */
+function fn_custom_carriers_get_carriers($is_custom)
```

Comment:
Enum YesNo используется как прослойка для хранения значений в БД, как значение одиночных параметров его лучше не использовать. Для преобразования булевых true|false в Y|N у enum'а есть метод toId.
```suggestion
 * @param bool $is_custom For select custom or not custom carriers
 *
 * @return array<string, array<string, string>> $normalize_carriers Normalize custom carriers
 */
function fn_custom_carriers_get_carriers($is_custom = true)
```